frontend/server/src/Controllers/Course.php

```diff
@@ -3075,6 +3080,15 @@ public static function getStudentProgressForSmarty(
             throw new \OmegaUp\Exceptions\ForbiddenAccessException();
         }
 
+        $studentsProgress = \OmegaUp\Cache::getFromCacheOrSet(
```

Comment:
nit:

```suggestion
        ['allProgress' => $studentsProgress] = \OmegaUp\Cache::getFromCacheOrSet(
```

así L3100 puede ser nada más `$studentsProgress`.